### PR TITLE
Update the CRI doc for 1.6

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

File: contributors/devel/container-runtime-interface.md [hunk 1]
```diff
@@ -88,6 +74,24 @@ besides discussions on Github issues.
 
 ## [Status update](#status-update)
 
+### Kubernetes v1.6 release (Docker-CRI integration Beta)
+ **The Docker CRI integration has been promoted to Beta, and been enabled by
+default in Kubelet**.
+  - **Upgrade**: It is recommended to drain your node before upgrading the
+    Kubelet. If you choose to perform in-place upgrade, the Kubelet will
+    restart all containers managed by Kuberntes on the node.
```
Comment (feiskyer) [inline contributors/devel/container-runtime-interface.md]: s/Kuberntes/Kubernetes

File: contributors/devel/container-runtime-interface.md [hunk 2]
```diff
@@ -26,59 +26,45 @@ pluggable container runtimes and build a healthier ecosystem.
...
 
-Start kubelet with the following flags:
-  - Use the Docker container runtime by `--container-runtime=docker`(the default).
-  - Enable CRI in kubelet by`--experimental-cri=true`.
-
-Please also see the [known issues](#docker-cri-1.5-known-issues) before trying
-out.
+We still maintain the old, non-CRI Docker integration, but it has been
+deprecated and scheduled to be removed in the next release (1.7). Please file
+a Github issue and include @kubernetes/sig-node for any CRI problem.
```
Comment (feiskyer) [inline contributors/devel/container-runtime-interface.md]: link to an existing team? e.g. `sig-node-bugs` or `sig-node-proposals`

Comment (dchen1107) [inline contributors/devel/container-runtime-interface.md]: +1

**Comment by dchen1107** [comment]
LGTM except nits above.

File: contributors/devel/container-runtime-interface.md (hunk 2 above)
Comment (yujuhong) [inline contributors/devel/container-runtime-interface.md]: fixed.

File: contributors/devel/container-runtime-interface.md (hunk 1 above)
Comment (yujuhong) [inline contributors/devel/container-runtime-interface.md]: done.

**Comment by yujuhong** [comment]
Addressed the comments. Applied lgtm based on https://github.com/kubernetes/community/pull/460#issuecomment-287112882

**Pull request merged**